Block/BlockWateringCan.cs

```diff
@@ -258,7 +264,7 @@ public override bool OnHeldInteractStep(float secondsUsed, ItemSlot slot, Entity
                 byEntity.World.SpawnParticles(WaterParticles, byPlayer);
             }
 
-            return true;
+            return base.OnHeldInteractStep(secondsUsed, slot, byEntity, blockSel, entitySel);

```

Comment:
this one also looks like it would break things

Reply:
BlockBehavior doesn't call OnHeldInteractStep because of this, which makes custom interactions impossible